### PR TITLE
Add landing page

### PR DESCRIPTION
**Pull request by MindPrepDev**
Here is the landing page (mobile version) for my personal portfolio site. This feature branch contains some linters files for efficiency evaluation, a completed static landing page (index.html file, css file, background image as and some icons with navigation links, with the exception of JavaScript and other DOM methods.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="mobile-size">
+  <img id="bg-img" src="images/Illustration-mobile.png" alt="background image" width="120">
+  <header>
+    <p class="list-style" id="header"><a href="https://www.github.com/MindPrepDev">Oluwatoyin Olaoye</a></p>
+    <nav>
+      <img src="images/icons8-menu-24.png" alt="menu list">
+    </nav>
+  </header>
```
Comment (mahbubul14) [inline index.html:17]: - [ ] Your design is not matched with the Figma design. Please work on it. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
y1 | f1

File: style.css [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+}
+
+body {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-end;
+  font-family: 'Roboto', sans-serif;
+  background-color: #1c1a19;
+  margin: 0;
+}
```
Comment (mahbubul14) [inline style.css:22]: - [ ] Your mobile version of the app is not currently optimized well. You can use [CSS Media Queries](https://www.w3schools.com/css/css3_mediaqueries.asp) to make your page responsive well. Please work on it. :writing_hand:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @MindPrepDev ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Please check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & descriptions.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @MindPrepDev,

CompletedCompletedItMateGIF


Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**